requirements_all.txt

```diff
@@ -2775,3 +2775,6 @@ zwave-js-server-python==0.49.0
 
 # homeassistant.components.zwave_me
 zwave-me-ws==0.4.3
+
+# homeassistant.components.zwave_me
```

Comment:
It seems like you manually wrote this?

Please use our `script.gen_requirements_all` to update generated files like this one.

Reply:
resolved